src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlCodeGeneration.cs

```diff
@@ -134,17 +134,17 @@ public XamlCodeGeneration(GeneratorExecutionContext context)
 
 			_historicalOutputGenerationPath = context.GetMSBuildPropertyValue("UnoXamlHistoricalOutputGenerationPath") is { Length: > 0 } value ? value : null;
 
-			var pageItems = GetWinUIItems("Page").Concat(GetWinUIItems("UnoPage"));
-			var applicationDefinitionItems = GetWinUIItems("ApplicationDefinition").Concat(GetWinUIItems("UnoApplicationDefinition"));
+			//var pageItems = GetWinUIItems("Page").Concat(GetWinUIItems("UnoPage"));
```

Comment:
I think we'll still need to get the actual files to use from the msbuild properties then match with the AdditionalFiles, so we keep getting the proper metadata.

Reply:
Yeah I figured that out. That's why I needed https://github.com/unoplatform/Uno.SourceGeneration/pull/152 to be able to use https://github.com/unoplatform/uno/blob/469a566ab18ff3179f0f54a2d9bfbeeca783f892/src/SourceGenerators/Uno.UI.SourceGenerators/Helpers/GeneratorExecutionContextExtensions.cs#L25-L34 for both uno and roslyn generations, so that I can access `AdditionalText` from `MSBuildItem` we define [here](https://github.com/unoplatform/uno/blob/469a566ab18ff3179f0f54a2d9bfbeeca783f892/src/SourceGenerators/Uno.UI.SourceGenerators/Helpers/GeneratorExecutionContextExtensions.cs#L39)